src/main/java/org/folio/service/budget/RolloverBudgetExpenseClassTotalsService.java

```diff
@@ -86,10 +88,13 @@ private BudgetExpenseClassTotal buildBudgetExpenseClassTotals(ExpenseClass expen
 
       MonetaryAmount tmpExpended = MoneyUtils.calculateTotalAmount(
         transactionGroupedByType.getOrDefault(Transaction.TransactionType.PAYMENT, Collections.emptyList()), currency);
-      tmpExpended = tmpExpended.subtract(MoneyUtils.calculateTotalAmount(
-        transactionGroupedByType.getOrDefault(Transaction.TransactionType.CREDIT, Collections.emptyList()), currency));
+      MonetaryAmount tmpCredited = MoneyUtils.calculateTotalAmount(
+        transactionGroupedByType.getOrDefault(Transaction.TransactionType.CREDIT, Collections.emptyList()), currency);
+
+      tmpExpended = tmpExpended.subtract(tmpCredited);
```

Comment:
You should not do that. Previously expended was `payment transactions - credit transactions`. Now it is simply `payment transactions`, and credited is `credit transactions`.

Reply:
thanks, I will fix that, regarding creditedPercentage, I didn't calculate like expendedPercentage that because we don't have totalCredit field. Is that correct?

Reply:
That's not good, we should have creditedPercentage too. You will have to add the calculation for totalCredited. A complication of doing that is that it is related to [MODFIN-372](https://folio-org.atlassian.net/issues/MODFIN-372). You should probably work on that too, to avoid confusion.

Reply:
it will be merged to **feature** branch and I have added as comment about this, and I will work and fix in scope of https://folio-org.atlassian.net/issues/MODFIN-372 ticket. For know, It is better to merge this ticket not to affect sprint. Fix will be another pr